### PR TITLE
added ability to add itinerary

### PR DESCRIPTION
**Pull request by aionate0812**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    dba:"postgres:localhost:5432/bundle",
+    DARKSKY_API_KEY:"aa9770871dcf844f5ca90bbdcd087e5b"
```
Comment (JoshuaKGoldberg) [inline config.js]: Just checking: these are intentionally checked in?

Comment (aionate0812) [reply config.js]: that is not supposed to be there, thanks for catching it

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+  - export DARKSKY_API_KEY=$DARKSKY_API_KEY
+  - export HEROKU_KEY=$HEROKU_KEY
+script:
+  - npm run config-file
+  - npm test
 deploy:
   provider: heroku
   api_key: $HEROKU_KEY
-  app: bundle-backend
+  app: mysterious-coast-95538
```
Comment (pamelaabreu) [inline .travis.yml]: Can we somehow change it to back to bundle-backend?

File: routes/itineraryRouter.js [hunk 3]
```diff
@@ -17,6 +16,33 @@ itineraryRouter.get('/:trip_id', (req, res) => {
...
+    const {name, address, phone, note, trip_id, itinerary_type} = req.body
+    itineraryService.addItinerary(name, address, phone, note, trip_id, itinerary_type)
+    .then( () => {
+        return itineraryService.getItinerary(trip_id) 
+    }, err => {
+        console.log(err)
+        res.json({msg:'Could not add itinerary'})
+    })
+    .then( itineraries => {
+        console.log(itineraries)
```
Comment (pamelaabreu) [inline routes/itineraryRouter.js]: These consoles can be cleaned up.

File: .gitignore [hunk 4]
```diff
@@ -1,3 +1,3 @@
 /node_modules
-/config.json
+/config.js
```
Comment (pamelaabreu) [inline .gitignore]: Are we going to replace the config.json with config.js?

**Pull request merged**